test/Sema/object_literals_osx.swift

```diff
@@ -6,8 +6,11 @@ struct S: _ExpressibleByColorLiteral {
 }
 
 let y: S = #colorLiteral(red: 1, green: 0, blue: 0, alpha: 1)
-let y2 = #colorLiteral(red: 1, green: 0, blue: 0, alpha: 1) // expected-error{{could not infer type of color literal}} expected-note{{import AppKit to use 'NSColor' as the default color literal type}}
-let y3 = #colorLiteral(red: 1, bleen: 0, grue: 0, alpha: 1) // expected-error{{incorrect argument labels in call (have 'red:bleen:grue:alpha:', expected 'red:green:blue:alpha:')}} expected-error{{could not infer type of color literal}} expected-note{{import AppKit to use 'NSColor' as the default color literal type}}
+let y2 = #colorLiteral(red: 1, green: 0, blue: 0, alpha: 1) // expected-error{{could not infer type of color literal}}
```

Comment:
I think we should keep it as is since this changes is unrelated to other tests.

Reply:
OK.
I will move it to new another patch.